src/ContinuumAbsorption/absorption_ff_positive_ion.jl

```diff
@@ -0,0 +1,74 @@
+include("Peach1970.jl")
+
+"""
+    positive_ion_ff_absorption!(α_out::Vector{Real}, ν::Real, T::Real, number_densities::Dict, ne::Real,
+                               departure_coefficients=Peach1970.departure_coefficients)
+
+Computes the free-free linear absorption coefficient (in cm⁻¹) for all species except H⁻ and He⁻.
+Uses provided departure coefficients when they available, and the uncorrected hydrogenic 
+approximation when they are not.
+
+# Arguments
+- `ν`: frequency in Hz
+- `T`: temperature in K
+- `number_densities` is a `Dict` mapping each `Species` to its number density.
+- `ne`: the number density of free electrons.
+- `departure_coefficients` (optional, defaults to [Peach1970](@ref)): 
+   a dictionary mapping species to the departure coefficients for the ff process it participates in 
+   (e.g. C II maps to the C III ff departure coefficients--see note below).  Departure coefficients 
+   should be callables taking temperature and (photon energy / RydbergH / Zeff^2).  See 
+   [Peach1970](@ref) for details.
+
+!!! note
+    A free-free interaction is named as though the species interacting with the free electron had 
+    one more bound electron (in other words it's named as though the free-electron and ion were 
+    bound together). The `Z` argument and `ni` values should be specified for the species that 
+    actually participates in the reaction. For example:
+    - Si I ff absorption: `ni` holds the number density of Si II, and `Z=1` (net charge of Si II)
+    - Si II ff absorption: `ni` holds the number density of Si III, and `Z=2` (net charge of Si III)
```

Comment:
```suggestion
!!! note
    A free-free interaction is named as though the species interacting with the free electron had 
    one more bound electron (in other words it's named as though the free-electron and ion were 
    bound together). To make matters more confusing, some sources present the free-free linear 
    absorption coefficient's formula in terms of the number density of the species in the 
    interaction's name (by including the Saha equation in the formula).

    To compute the absorption for a given free-free interaction, this function accesses elements 
    from the dictionary arguments that are associated with the species that participates in the 
    interaction. For example:
    - Si I ff absorption: uses the number density of Si II (`number_densities[species"Si II"]`) and
      checks `departure_coefficients[species"Si II"]` for departure coefficients.
    - Si II ff absorption: uses the number density of Si III (`number_densities[species"Si III"]`)
      and checks `departure_coefficients[species"Si III"]` for departure coefficients.
```
This topic is confusing and I think it may help to add further clarification like the above